docs/_templates/footer.html

```diff
@@ -19,4 +50,20 @@
     </a>
     {% endif %}
 
+  <div class="sponsors">
```

Comment:
This looks pretty ugly, but we can iterate.

Might be worth using a class name with less propensity for blocking? `supporters`?